src/foam/dashboard/view/GroupByCitationView.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.CLASS({
+  package: 'foam.dashboard.view',
+  name: 'GroupByCitationView',
+  extends: 'foam.dashboard.view.DashboardCitationView',
+  mixins: ['foam.nanos.controller.MementoMixin'],
+
+  documentation: `
+    A citation view that displays a property value and the number of dao entries that share the same value.
+  `,
+
+  imports: [
+    'translationService'
+  ],
+
+  requires: [
+    'foam.comics.v2.DAOBrowseControllerView',
+    'foam.u2.stack.StackBlock'
+  ],
+
+  implements: [
+    'foam.mlang.Expressions'
+  ],
+
+  properties: [
+    {
+      class: 'String',
+      name: 'groupByPropertyName'
+    },
+    {
+      class: 'FObjectProperty',
+      of: 'foam.nanos.menu.Menu',
+      name: 'redirectMenu'
+    },
+    'dao',
+    'customizeKey'
+  ],
+
+  methods: [
+    async function render() {
+      this.initMemento();
+
+      var self = this;
+
+      var label = this.customizeKey ? await this.customizeKey(this.data[0]) : this.data[0];
+      var count = this.data[1];
+
+      this.addClass(this.myClass())
+        .on('click', function() {
```

Comment:
Add `nodeName: 'button'` prop to this view so that it is keyboard focusable along with aria labels,  otherwise you can move this click to an action and use an unstyled ActionView to wrap the whole row